hugolib/shortcode.go

```diff
@@ -460,6 +458,10 @@ Loop:
 		switch {
 		case currItem.IsLeftShortcodeDelim():
 			next := pt.Peek()
+			if next.IsRightShortcodeDelim() {
+				// no name: {{< >}} or {{% %}}
+				return sc, errors.New("Shortcode has no name")
```

Comment:
Go errors should start with lowercase; we probably don't do that everywhere, but I see all the errors in this file does, so we might as well be lenient about this.

Reply:
Thanks. I will be careful from now on